### PR TITLE
CASMHMS-6281 imgstat

### PR DESCRIPTION
**Pull request by rustydb**
Adds `imgstat` command call after the `initrd` command in order to print image information such as image size. This can help diagnose image download issues such as [CASMTRIAGE-7220](https://jira-pro.it.hpe.com:8443/browse/CASMTRIAGE-7220).

Also updates `.gitignore` to cease this annoying warning:
```
git add cmd/boot-script-service/default_api.go
The following paths are ignored by one of your .gitignore files:
cmd/boot-script-service
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"
```

`.gitignore` test:
```bash
[534]rusty@HPE-XHD22YD7DW:~/gitstuffs/cray-shasta/hms-bss> echo 1>boot-script-service
[535]rusty@HPE-XHD22YD7DW:~/gitstuffs/cray-shasta/hms-bss> git add boot-script-service
The following paths are ignored by one of your .gitignore files:
boot-script-service
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"
[536]rusty@HPE-XHD22YD7DW:~/gitstuffs/cray-shasta/hms-bss> echo 1>>cmd/boot-script-service/default_api.go
[537]rusty@HPE-XHD22YD7DW:~/gitstuffs/cray-shasta/hms-bss> git add cmd/boot-script-service/default_api.go
[538]rusty@HPE-XHD22YD7DW:~/gitstuffs/cray-shasta/hms-bss>
```

### CODE REVIEW

**Comment by rustydb** [comment]
Doesn't look like unit tests can even run in CI anymore.

File: cmd/boot-script-service/default_api.go [hunk 1]
```diff
@@ -628,21 +628,23 @@ func buildBootScript(bd BootData, sp scriptParams, chain, role, subRole, descr s
...
+        if err == nil {
+            script += "initrd --name initrd " + u + " || goto boot_retry\n"
+            script += "imgstat || echo Could not show image information."
+        }
+    }
+    script += "boot || goto boot_retry\n:boot_retry\n"
+    // We could vary the length of the sleep based on retry count or some
+    // other criteria.
+    // For now, just sleep a bit
+    script += fmt.Sprintf("sleep %d\n", retryDelay) + chain + "\n"
```
Comment (rustydb) [inline cmd/boot-script-service/default_api.go:647]: tabs vs. spaces issue

Comment (rustydb) [reply cmd/boot-script-service/default_api.go]: Fixed in both commits.

**Comment by jwlv** [comment]
> Doesn't look like unit tests can even run in CI anymore.

@rustydb I ran into this recently too in a different repo.  We need to change "docker-compose" to "docker compose".  See this PR from last week on what I did:  https://github.com/Cray-HPE/hms-smd/pull/163

**Comment by jwlv** [comment]
> > Doesn't look like unit tests can even run in CI anymore.
> 
> @rustydb I ran into this recently too in a different repo. We need to change "docker-compose" to "docker compose". See this PR from last week on what I did: [Cray-HPE/hms-smd#163](https://github.com/Cray-HPE/hms-smd/pull/163)

**Review by jwlv — APPROVED** [review]

**Pull request merged**